certbot/storage.py

```diff
@@ -256,8 +259,26 @@ def __init__(self, config_filename, cli_config):
         self.live_dir = os.path.dirname(self.cert)
 
         self._fix_symlinks()
+        if update_symlinks:
+            self._update_symlinks()
         self._check_symlinks()
 
+    def __str__(self):
+        """Returns a string representation of the object."""
+        # import ipdb; ipdb.set_trace()
+        outdict = {"Lineage": self.lineagename,
+                   "Domains": " ".join(self.names())}
+        return "RenewableCert(%s)" % str(outdict)
```

Comment:
If this is used as the basis for the `_report()`, which apparently it is, the user probably won't need to see the text "RenewableCert" (which is more of a Certbot internal thing and would probably be confusing to the user). Conversely, I think it would be useful to include the expiration date in the output because that's something users commonly care about intensely.